test/tpch/mysql-bench/conf.d/my.cnf

```diff
@@ -0,0 +1,2 @@
+[mysqld]
+innodb_buffer_pool_size=2G
```

Comment:
Improves query performance allowing more memory, required for SF1 scale to complete successfully on top of indexes